frontend/src/components/matching/Countdown.tsx

```diff
@@ -43,9 +39,11 @@ function Countdown() {
     }
 
     console.log(`API update call made`);
-    const checkRequest: CheckQueueStatusRequest = { sessionToken };
+    const checkRequest: CheckQueueStatusRequest = {};
     axios
-      .post<CheckQueueStatusResponse>("/queue/status", checkRequest)
+      .post<CheckQueueStatusResponse>("/queue/status", checkRequest, {
```

Comment:
Since we are passing in an empty object, maybe we don't need to initialise `checkRequest`
```suggestion
      .post<CheckQueueStatusResponse>("/queue/status", {}, {
```

Reply:
Fixed in commit [b0449fa](https://github.com/CS3219-AY2223S1/cs3219-project-ay2223s1-g33/pull/95/commits/b0449fa73d19964b07579416189991ff91fa6c45)